app/views/shared/_cohort.haml

```diff
@@ -0,0 +1,6 @@
+%li= link_to "All", root_path
+- cohorts.each do |cohort|
+  - if cohort.upcase == ENV['CLASS_ONE']
```

Comment:
Yes - but then how do we know which once was chosen as the first one and which one was chosen as the second? By having the actual class name in the ENV variable, then I don't think there is a way to confuse it (to match what they fill out in the form). Maybe i'm wrong though.